### PR TITLE
JBIDE-19903
add javaee site so that these tes...
...ts can be run against upstream site, rather than expecting dependencies to have been built & resolved locally

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

File: batch/itests/pom.xml [hunk 1]
```diff
@@ -17,6 +17,21 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 		<module>org.jboss.tools.batch.core.itest</module>
 		<module>org.jboss.tools.batch.ui.itest</module>
 	</modules>
+
+	<!-- JBIDE-10933 add javaee site so that these tests can be run against upstream site, rather than expecting dependencies to have been built & resolved locally -->
```
Comment (alexeykazakov) [inline batch/itests/pom.xml]: Wrong JIRA btw. Should be 19903.

Comment (dgolovin) [inline batch/itests/pom.xml]: That is not enough though, it also should tell to tycho to ignore what is in local repository, to be sure nothing is picked up that was built locally.

**Comment by nickboldt** [comment]
Comment JIRA number fixed.

As to needing to prevent use of locally-built stuff... why should that matter? Whatever's newer should be used -- either the remote stuff or the local stuff.

BTW, I ran the matrix job with the PR fetched from Jenkins and 2 children were 2, one yellow, and one grey (timed out after 2hrs). Running again w/ 3hr timeout and revised PR:

https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/jbosstools-javaee-tests-matrix_master/ >=6

File: batch/itests/pom.xml [hunk 2]
```diff
@@ -17,6 +17,21 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
...
 		<module>org.jboss.tools.batch.ui.itest</module>
 	</modules>
+
+	<!-- JBIDE-19903 add javaee site so that these tests can be run against upstream site, rather than expecting dependencies to have been built & resolved locally -->
+	<profiles>
+		<profile>
+			<id>runTestsOnly</id>
+			<repositories>
+				<repository>
+					<id>jbosstools-javaee</id>
```
Comment (maxandersen) [inline batch/itests/pom.xml]: why is this done only for tests? this seems very very wrong.

**Comment by maxandersen** [comment]
i'm not following what the goal is here but if intent is to make sure it uses what is on the updatesites then Denis comment is correct as otherwise it will just take what is available locally that you might built in a separate branch/PR.

**Comment by nickboldt** [comment]
re: Denis' comment... as I already said above... 

"As to needing to prevent use of locally-built stuff... why should that matter? Whatever's newer should be used -- either the remote stuff or the local stuff."

So there's no need to filter out anything built locally. 

If using this profile locally, you could of course do that with `-Dtycho.localArtifacts=ignore` if you wanted to, but again, there's no need.

**Pull request merged**